multi-node/aws/pkg/config/config.go

```diff
@@ -136,6 +136,8 @@ type Cluster struct {
 	KMSKeyARN                string            `yaml:"kmsKeyArn,omitempty"`
 	CreateRecordSet          bool              `yaml:"createRecordSet,omitempty"`
 	RecordSetTTL             int               `yaml:"recordSetTTL,omitempty"`
+	TLSCADurationDays        int               `yaml:"tlsCAValidDurationDays,omitempty"`
```

Comment:
Oh yeah, missed updating the yaml tag when I was playing with names. 

Right now I validate the values in `command_render.go` but we can add test validation as well. Also `x509.go` additionally will use default values if it receives durations <= 0.

I'll change `x509.go` do the runtime validation but not set defaults. Add a test validation step. And set the defaults the normal way instead.